src/planscape/planscape/settings.py

```diff
@@ -108,6 +108,9 @@
 
 WSGI_APPLICATION = "planscape.wsgi.application"
 
+# TODO: This is for debugging email messages, and it needs to be removed.
+EMAIL_BACKEND='django.core.mail.backends.console.EmailBackend'
```

Comment:
According to the TODO message this needs to be removed or restructured into a settings.dev.py file.

Reply:
Rebasing to main should resolve this todo.
The TODO was resolved here: https://github.com/OurPlanscape/Planscape/pull/948